Update notifications article (#19250)
newrelic/docs-website: src/content/docs/codestream/codestream-settings/account-settings.mdx

---
title: CodeStream notifications
metaDescription: "An overview of CodeStream notifications."
freshnessValidatedDate: 2024-11-08
---

> CodeStream sends email notifications when new [transaction-level anomalies](/docs/codestream/observability/transactions) are found in services that are built from your code. Select the <DNT>**Notifications**</DNT> option under the username menu at the top of the CodeStream pane to control which repositories and services you'll be notified about, or to turn off notifications completely.

Reviewer: ```suggestion
CodeStream alerts you via email when new [transaction-level anomalies](/docs/codestream/observability/transactions) are detected in services built from your code. To manage these notifications, go to the <DNT>**Notifications**</DNT> settings under your user profile. You can customize which repositories and services trigger notifications or disable them entirely.
```

> By default, you will automatically be following any repository that you open in your IDE. You can alternatively decide to manually follow repositories. Under this model you can follow a repository by opening it in your IDE, then hovering over its entry in the CodeStream tree view and clicking on the Follow icon.

Reviewer: ```suggestion
By default, CodeStream automatically tracks repositories you open in your IDE. Alternatively, you can manually follow repositories by hovering over them in the CodeStream tree view and clicking the **Follow** icon.
```

<img
  title="Manually following a repository."
  alt="A screenshot of the icon for following a repository."
  src="/images/codestream_screenshot-crop_follow-repo.webp"
/>

> You can unfollow a repository by visiting the <DNT>**Notification Settings**</DNT> page and clicking the "X" to the right of any repository that you're current following.

Reviewer: ```suggestion
You can unfollow a repository by visiting the <DNT>**Notification Settings**</DNT> page and clicking the **X** to the right of any repository that you're current following.
```

Reviewer: To confirm, is it **Notification Settings** or **Notifications** ?

> By default, you will be notified about anomalies in any of the services associated with the repositories that you're following. You may, however, only want to get notifications for a subset of these services. For example, you might only want notifications for your production services. You can limit the services by specifying a keyword present in the service name (e.g., "prod"), a list of account IDs, or a set of key:value pairs (e.g., environment:production).

Reviewer: ```suggestion
By default, you'll receive notifications for anomalies in all services associated with your followed repositories. To limit notifications to specific services, you can filter by service name (for example, "prod"), account ID, or key-value pairs (such as `environment:production`).
```

<img

>   title="Notification settings."

Reviewer: ```suggestion
  title="Notification settings"
```
